components/overledger/overledger.app.mjs

```diff
@@ -22,24 +22,42 @@ export default {
       };
     },
     _makeRequest({
-      $ = this, path, ...otherOpts
+      $ = this, baseUrl, path, ...otherOpts
     }) {
       return axios($, {
         ...otherOpts,
-        url: this._baseUrl() + path,
+        url: baseUrl + path,
```

Comment:
I'm not that familiar with this type of code use, but it seems like it would work as intended, that is, a default is used unless another overwriting parameter is provided

Reply:
ok lets add this code then